components/Staking.tsx

```diff
@@ -38,19 +38,17 @@ const Staking: React.FC = () => {
       )
       .map((balance) => {
         const bal = balance;
-        // @ts-expect-error bufs for the win
-        (
-          bal.balanceView?.valueView?.value as ValueView_KnownAssetId
-        ).metadata.symbol = truncateMiddle(
-          // @ts-expect-error bufs for the win
-          (bal.balanceView?.valueView?.value as ValueView_KnownAssetId).metadata
-            .symbol,
+
+        (bal.balanceView?.valueView?.value as ValueView_KnownAssetId)!
+          .metadata!.symbol = truncateMiddle(
```

Comment:
question: did you check if there was a getter in `/web` for getting symbols? If so you should use that.

Reply:
The symbol truncation here is for cosmetic reasons. I did not find anything like this in the web repo upon a brief look, happy to be proven wrong tho.

Reply:
Nah, was thinking something more like this for getting the symbol from the buf type: https://github.com/penumbra-zone/web/blob/074584500b1dd143c927cadb8ceb85d0b72e5204/packages/getters/src/value-view.ts#L68

Reply:
Ahhh I see! Thanks for the suggestion, fixed.